developer/src/kmc/src/util/getLastGitCommitDate.ts

```diff
@@ -1,7 +1,7 @@
 import { execFileSync } from 'child_process';
 
 // RFC3339 pattern, UTC
-export const expectedGitDateFormat = /^\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\dZ$/;
+export const expectedGitDateFormat = /^\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d(\.\d\d\d)?Z$/;
```

Comment:
Does this impact l.9 on the type of date /time string? Is it an optional milliseconds?

Reply:
yeah comment should be updated...